src/main/java/duke/command/ByeCommand.java

```diff
@@ -0,0 +1,24 @@
+package duke.command;
+
+/**
+ * ByeCommand is a command for Duke to terminate.
+ *
+ * @author totsukatomofumi
+ */
+public class ByeCommand extends Command {
+    /**
+     * Constructor for the command.
+     */
+    public ByeCommand() {
+        super();
+    }
+
+    /**
+     * Terminates the program.
+     */
+    @Override
+    public void run() {
+        System.out.println("Bye. Hope to see you again soon!");
```

Comment:
Should the print statements be moved to the UI class, since they deal with the user ?
Noticed the same thing with the other commands